### PR TITLE
Adds DustLog History to Binance and adds additional response to Kraken.

### PR DESCRIPTION
**Pull request by gregoreficint**
Adds new ability to get DustLog history from Binance. See https://github.com/knowm/XChange/issues/4371

Adds the new response "EGeneral:Too many requests" from Kraken to the base service. See https://github.com/knowm/XChange/issues/4369#issue-1088385658

### CODE REVIEW

File: xchange-binance/src/main/java/org/knowm/xchange/binance/service/BinanceTradeService.java [hunk 1]
```diff
@@ -228,6 +229,27 @@ public boolean cancelOrder(CancelOrderParams params) throws IOException {
...
+   * Balance to BNB".
+   *
+   * @param startTime optional. If set, also the endTime must be set. If neither time is set, the
+   *     100 most recent dust logs are returned.
+   * @param endTime optional. If set, also the startTime must be set. If neither time is set, the
+   *     100 most recent dust logs are returned.
+   * @return
+   * @throws IOException
+   */
+  public BinanceDustLog getDustLog(Long startTime, Long endTime) throws IOException {
```
Comment (walec51) [inline xchange-binance/src/main/java/org/knowm/xchange/binance/service/BinanceTradeService.java:245]: this method should be implemented in `BinanceTradeServiceRaw`

File: xchange-binance/src/main/java/org/knowm/xchange/binance/service/BinanceTradeService.java [hunk 2]
```diff
@@ -228,6 +229,27 @@ public boolean cancelOrder(CancelOrderParams params) throws IOException {
...
+
+    if (((startTime != null) && (endTime == null)) || (startTime == null) && (endTime != null))
+      throw new ExchangeException("You need to specify both, the start and the end date");
```
Comment (walec51) [inline xchange-binance/src/main/java/org/knowm/xchange/binance/service/BinanceTradeService.java:248]: You need to specify both the start and the end date or none of them

File: xchange-binance/src/main/java/org/knowm/xchange/binance/service/BinanceTradeService.java (hunk 1 above)
Comment (gregoreficint) [reply xchange-binance/src/main/java/org/knowm/xchange/binance/service/BinanceTradeService.java]: Done

File: xchange-binance/src/main/java/org/knowm/xchange/binance/service/BinanceTradeService.java (hunk 2 above)
Comment (gregoreficint) [reply xchange-binance/src/main/java/org/knowm/xchange/binance/service/BinanceTradeService.java]: Done

**Pull request merged**